.github/workflows/ci.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
   pull_request:
   release:
-    types: [ released ]
+    types: [released]
```

Comment:
Side note: it's probably wise we add a Prettier config to this repo at some stage.

Reply:
Good point. I didn't know that it supported YAML. Did you apply it here for this PR?

Reply:
My IDE did it automatically.